packages/indy-sdk/src/anoncreds/services/IndySdkAnonCredsRegistry.ts

```diff
@@ -509,6 +511,47 @@ export class IndySdkAnonCredsRegistry implements AnonCredsRegistry {
       }
     }
   }
+
+  public async fetchIndySchemaWithSeqNo(agentContext: AgentContext, seqNo: number, did: string) {
+    const indySdkPoolService = agentContext.dependencyManager.resolve(IndySdkPoolService)
+    const indySdk = agentContext.dependencyManager.resolve<IndySdk>(IndySdkSymbol)
+
+    const { pool } = await indySdkPoolService.getPoolForDid(agentContext, did)
+    agentContext.config.logger.debug(`Getting transaction with seqNo '${seqNo}' from ledger '${pool.didIndyNamespace}'`)
+
+    const request = await indySdk.buildGetTxnRequest(did, 'DOMAIN', seqNo)
+
+    agentContext.config.logger.trace(`Submitting get transaction request to ledger '${pool.didIndyNamespace}'`)
+    const response = await indySdkPoolService.submitReadRequest(pool, request)
+
+    const schema = response.result.data as SchemaType
+
+    const schemaId = getLegacySchemaId(did, schema.txn.data.data.name, schema.txn.data.data.version)
+
+    return {
+      schema: {
+        schemaId,
+        attr_name: schema.txn.data.data.attr_names,
+        name: schema.txn.data.data.name,
+        version: schema.txn.data.data.version,
+        issuerId: did,
+        seqNo,
+      },
+      indyNamespace: pool.didIndyNamespace,
+    }
+  }
+}
+
+export interface SchemaType {
```

Comment:
I don't think there's a need to export this interface, it's only used internally

```suggestion
interface SchemaType {
```